internal/contract/types.go

```diff
@@ -102,6 +102,42 @@ func (i *Int64) Set(obj *unstructured.Unstructured, value int64) error {
 	return nil
 }
 
+// Int32 represents an accessor to an int32 path value.
+type Int32 struct {
+	paths []Path
+}
+
+// Path returns the path to the int64 value.
```

Comment:
and converts it to int32?

Reply:
Nice catch, fixed all the comments about this type